src/fiber/fiber.mli

```diff
@@ -63,6 +63,8 @@ val both : 'a t -> 'b t -> ('a * 'b) t
     the [Applicative] interface.*)
 val all : 'a t list -> 'a list t
 
+val all_concurrently_unit : unit t list -> unit t
```

Comment:
Could you move this definition next to `all_concurrently`? This part of the API is for sequential operations.